src/styles/index.scss

```diff
@@ -1,3 +1,148 @@
+  "blue-accent": #00acdc,
+  "gray-element": #f3f3f3,
+  "yellow-stars": #ffde6a
+);
+
+$sizes: (
+  "star-size": 16px,
+  "container-size": 96px,
+);
+
+/* #region card */
+
 body {
   margin: 0;
 }
+
+.card {
+  height: 406px;
+  width: min-content;
+  border: 1px solid map-get($colors, "gray-element");
+  border-radius: 5px;
+  background-color: map-get($colors, "white");
+  font-family: "Roboto", sans-serif;
+  &__rectangle {
+    width: 10rem;
+    padding: 2rem 19px 0;
+    margin-bottom: 36px;
+    height: 134px;
+  }
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: map-get($colors, "main-accent");
+    padding: 0 1rem;
+    margin-bottom: 0.25rem;
+  }
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: map-get($colors, "secondary");
+    padding: 0 1rem;
+    margin-bottom: 1rem;
+  }
+  &__review {
+    padding: 0 1rem;
+    margin-bottom: 1.5rem;
+  }
+  &__button {
+    padding: 0 1rem 1rem;
+    display: flex;
+    justify-content: center;
+  }
+  &__price {
+    padding: 0 1rem;
+    margin-bottom: 1rem;
+  }
+}
+
+.review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  &__stars {
+    width: 96px;
+  }
+  &__text {
+    color: map-get($colors, "main-accent");
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+  &__price {
+    padding: 0 1rem;
+    margin-bottom: 1rem;
+  }
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  &__text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: map-get($map: $colors, $key: secondary);
+  }
+  &__value {
+    font-weight: 700;
+    line-height: 18px;
+    color: map-get($colors, "main-accent");
+  }
+}
+
+.button {
+  text-decoration: none;
+  display: inline-block;
+  background-color: map-get($colors, "blue-accent");
+  color: map-get($colors, "white");
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  border: 1px solid map-get($colors, "blue-accent");
+  border-radius: 5px;
+  width: 166px;
+  height: 2.5rem;
+  &:hover {
+    background-color: map-get($colors, "white");
+    color: map-get($colors, "blue-accent");
+    border: 1px solid map-get($colors, "blue-accent");
+    border-radius: 5px;
+  }
+}
+
+/* #endregion */
+
+/* #region stars */
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  &__star {
+    background-image: url(..\images\star.svg);
+    height: map-get($map: $sizes, $key:star-size);
+    width: map-get($map: $sizes, $key:star-size);
+    background-position: center;
+    object-fit: cover;
+    box-sizing: border-box;
+    background-repeat: no-repeat;
+  }
+  &--5 .stars__star {
+    background-image: url(\images\star-active.svg);
+  }
+}
+
+@for $index from 1 through 4 {
+  .stars--#{$index} .stars__star:nth-child(-n + #{$index}) {
+    background-image: url(..\images\star-active.svg);
+  }
+}
```

Comment:
Could you handle it just with the loop? Since the loop is responsible for setting star-active I would remove the `&--5 .stars__star` selector